cloudify_cli/commands/users.py

```diff
@@ -113,17 +113,27 @@ def list(sort_by,
 @cfy.options.verbose()
 @cfy.options.security_role
 @cfy.options.password
+@cfy.options.tenant_name(required=False)
+@cfy.options.user_tenant_role(required=False,
+                              options_flags=['-l', '--user-tenant-role'])
 @cfy.assert_manager_active()
 @cfy.pass_client()
 @cfy.pass_logger
-def create(username, security_role, password, logger, client):
+def create(username, security_role, password, tenant_name, user_tenant_role,
+           logger, client):
```

Comment:
if func definition is too long for 1 line, better look like this:
func(arg1,
        arg2,
        arg3):